fixed ignore_errors during copy files for relative paths

jeviolle opened the pull request:
ignore_errors would never be evaluated for roles using copy. This however works fine for standard play books. output examples below.

https://gist.github.com/jeviolle/202274c74247c12efe4f

Commit:
fixed ignore_errors during copy files for relative paths

Commit:
fixed ignore_errors during copy files for relative paths

mpdehaan commented:
Can you please share the error before that was not being ignored and the playbook that reproduces it?

Some things with ignore_errors we do want to continue to be fatal, such as missing sources.

jeviolle commented:
This gist shows an example playbook/role and the output

https://gist.github.com/jeviolle/202274c74247c12efe4f

weargoggles committed:
Remove no-op string declaration

willthames committed:
Use https for git submodules

It's much more likely that people can use the https protocol
than the git protocol (many firewalls block the latter)

Commit:
Inventory Plugin to get hostgroups from FreeIPA

batmat committed:
Typo: recieve -> receive.

mpdehaan committed:
Merge pull request #9328 from Batmat/fix-receive-typo

Typo: recieve -> receive.

mpdehaan committed:
Merge pull request #9299 from weargoggles/quoted-comment

Remove no-op string declaration

mpdehaan committed:
Merge pull request #9303 from willthames/gitmodules_over_https

Use https for git submodules

mpdehaan committed:
Merge pull request #9312 from admiyo/freeipa

Inventory Plugin to get hostgroups from FreeIPA

mpdehaan committed:
Merge pull request #9331 from marcinpraczko/devel

Fix detecting distribution release on OpenSuSE

Commit:
Merge branch 'devel' of github.com:jeviolle/ansible into devel
